content/en/platform/corda/5.0-beta/corda-cli-reference/cli-commands/rpc-permission-templates.md

```diff
@@ -0,0 +1,43 @@
+---
+date: '2023-01-10'
+menu:
+  corda-5-beta:
+    identifier: corda-5-beta-rpc-permissions-templates
+    weight: 6000
+    parent: corda-5-cli-reference
+section_menu: corda-5-beta
+title: "RPC Permission Templates"
+---
+
+
+By default, when a cluster starts there is a single "admin" account auto-created which can do everything. While this account can be used to perform any action, there is room for error if not used carefully. RPC permission templates enable you to create fine-grained roles for specific actions such as:
+
+* A dedicated role which can create users, roles, permissions and drive all the associations between them.
+* A dedicated role with a set of all the necessary permissions to create a virtual node (including CPI upload).
+* A dedicated role which allows flows to run on this virtual node.
+
+{{< note >}}
+These roles and permissions enable certain common operations. The creation of users and associating such users to these roles should be done manually by an admin user.
+{{< /note >}}
+
+| Role   | Description |
```

Comment:
The descriptions aren't clear. If I understand correctly, each is a role that grants some permissions to a user? So the descriptions should all start with something like:
"Allows a user to... "
or
"Grants the necessary permissions to..."